### PR TITLE
add Spring tools

### PR DESCRIPTION
**Pull request by Bogdan-Demianchuk**

### CODE REVIEW

File: src/main/java/config/AppConfig.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+        Properties properties = new Properties();
+        properties.put("hibernate.dialect", "org.hibernate.dialect.MySQL8Dialect");
+        properties.put("hibernate.show_sql", env.getProperty("hibernate.show_sql"));
+        properties.put("hibernate.hbm2ddl.auto", env.getProperty("hibernate.hbm2ddl.auto"));
+
+        factoryBean.setHibernateProperties(properties);
+        factoryBean.setAnnotatedClasses(CinemaHall.class, Movie.class,
+                MovieSession.class, Order.class,
+                ShoppingCart.class, Ticket.class, User.class);
```
Comment (dmytro-hryhoruk) [inline src/main/java/config/AppConfig.java:56]: let's use sePackagesToScan()

File: src/main/java/config/WebConfig.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import org.springframework.web.servlet.view.InternalResourceViewResolver;
+import org.springframework.web.servlet.view.JstlView;
+
+@Configuration
+@EnableWebMvc
+@ComponentScan("com.cinema.controller")
+public class WebConfig {
+
+    @Bean
+    public InternalResourceViewResolver getResolver() {
```
Comment (dmytro-hryhoruk) [inline src/main/java/config/WebConfig.java:16]: if you don't have .jsp in your project, than we can remove this view resolver) and dependencies which are related to jsp from pom.xml

File: pom.xml [hunk 3]
```diff
@@ -38,6 +39,52 @@
...
+        <dependency>
+            <groupId>javax.servlet.jsp.jstl</groupId>
+            <artifactId>javax.servlet.jsp.jstl-api</artifactId>
+            <version>1.2.1</version>
+        </dependency>
+        <dependency>
+            <groupId>taglibs</groupId>
+            <artifactId>standard</artifactId>
+            <version>1.1.2</version>
+        </dependency>
```
Comment (dmytro-hryhoruk) [inline pom.xml:77]: do you need these dependencies?

File: src/main/java/config/AppConfig.java (hunk 1 above)
Comment (Bogdan-Demianchuk) [reply src/main/java/config/AppConfig.java:56]: Done

File: src/main/java/config/WebConfig.java (hunk 2 above)
Comment (Bogdan-Demianchuk) [reply src/main/java/config/WebConfig.java:16]: Done

File: pom.xml (hunk 3 above)
Comment (Bogdan-Demianchuk) [reply pom.xml:77]: No, Deleted

**Pull request merged**